python_backend/api/v1/router.py

```
@@ -0,0 +1,43 @@
+from api.v1.endpoints.conversion import router as conversion_router
+from api.v1.endpoints.user import router as user_router
+
+api_router=APIRouter()
+# v1전체의 라우팅 집합 
+# v1 대표 라우터
+
+
+#health 파일 내 router=APIRouter() 로 선언된 라우터 객체
+
+api_router.include_router(
+    health.router,
```

Ruff (0.12.2)

Undefined name `health`

(F821)